webapi/scripts/KnoraRules.pie

```diff
@@ -308,6 +308,13 @@ Id: maxCardinality_1_table
  * Consistency rules for subject and object class constraints.
  */
 
+// Every subproperty of knora-base:resourceProperty must have a knora-base:objectClassConstraint.
+Consistency: object_class_constraint_required
+    p <rdfs:subPropertyOf> <knora-base:resourceProperty> [Constraint p != <knora-base:resourceProperty>]
```

Comment:
Could we add a similar check for other property predicates such as GUI element? If such information is missing, the GUI does not work correctly.